dts/arm/nxp/nxp_rw6xx_common.dtsi

```diff
@@ -515,11 +516,16 @@
 		status = "disabled";
 	};
 
+	nbu: nbu {
+		compatible = "nxp,nbu";
+		/* first index is the nbu RX interrupt */
+		/* the second is the wake up done interrupt */
```

Comment:
nit but these comments are really not necessary, I can see from the `interrupt-names`, just remove